doc/system_requirements.md

```diff
@@ -305,9 +305,125 @@ Needs: dsn
 #### Get Changed Dependency
 `req~npm-changed-dependency~1`
 
-PK can retrieve changed NPM dependencies that where added, updated or removed since the last release.
+PK can retrieve changed NPM dependencies that were added, updated or removed since the last release.
 
 Covers:
 * `feat~npm-project-support~1`
 
 Needs: dsn
+
+### Automatic Dependency Update Process
+`feat~automatic-dependency-update-process~1`
+
+PK supports the automatic dependency process. This speeds up fixing vulnerabilities in third party dependencies and creating releases.
+
+Rationale:
+
+We have 130+ projects in the integration team that often require dependency updates due to security issues that are found in the dependencies or transitive dependencies. Most of the time the update is a combination of pulling the latest source, updating the dependencies, updating the change log, running the tests locally, on success pushing the branch, running CI and creating a release.
+
+* Auto-update dependencies
+* Automatically create change log entry
+* Automatically run local tests
+* Automatically push branch
+* Automatically run CI
+* Release
```

Comment:
```suggestion
* Update dependencies
* Create change log entry
* Run local tests
* Push branch
* Run CI
* Release
```

As the last item "release" is the only one not prefixed by "automated": Should we maybe add a "manually" or "partially manually" here? Maybe a more specific comment would be enlightening?

Reply:
Also in the headlines of the following sections we could remove the redundant "auto" and "automated".